yar/yar.php

```diff
@@ -1,5 +1,30 @@
 <?php
 
+
+/**
+ * The constants below are defined by this extension,
+ * and will only be available when the extension has either been compiled into PHP or dynamically loaded at runtime.
+ * @link https://secure.php.net/manual/en/yar.constants.php
+ */
+define('YAR_VERSION', '	2.0.5', true);
```

Comment:
These constants are not case insensitive, so,please, get rid of the third argument

Reply:
I fixed the problem